lookout/style/typos/utils.py

```diff
@@ -22,6 +23,8 @@
     ("confidence", float),  # Model confidence for the correction
 ))
 
+TEMPLATE_DIR = os.path.join(os.path.dirname(__file__), "templates")
```

Comment:
I believe it is not in `__init__.py` as we have in `lookout.style.format` because of some import problems, right?

Reply:
Circular import.
In format analyzer, it's solved in configuration https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/config.py#L75
So I'm not sure if it's used anywhere https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/__init__.py#L8

Reply:
yeah, looks like we should do something similar in style-analyzer.